test/yaml-rest-runner/src/main/java/org/elasticsearch/test/rest/yaml/ClientYamlTestExecutionContext.java

```diff
@@ -213,6 +214,24 @@ public Version esVersion() {
         return clientYamlTestClient.getEsVersion();
     }
 
+    /**
+     * @return the version that skipping tests is based on, usually the same as {@link #esVersion()}
+     */
+    public Version skipVersion() {
```

Comment:
I would not call this skipVersion, but rather getEsVersion like in the test client. For the multi_cluster tests, the es version will take into account the remote cluster.